### DISCUSSION TITLE
Added a hot bed to my finder, can I still use this plugin?

### DISCUSSION THREAD

**Discussion by Bazmundi** [ronoaldo/FlashforgeFinderIntegration#22] [General]
Thanks for producing the plugin!  I am sure many people are appreciative.

Long time Finder and FlashPrint boffin.   But now I am getting dangerous, as I want to extend my knowledge.

Gist is at [here](https://organicmonkeymotion.wordpress.com/2022/08/17/hot-plates-for-finders-part-1/), then [here](https://organicmonkeymotion.wordpress.com/2022/10/21/hot-sizzling/), and then [here](https://organicmonkeymotion.wordpress.com/2022/11/12/too-tall/).

The basic story, I bought a second removable bed for my finder and mounted a hot plate - call me crazy :zany_face:.   Plenty of thought given as to how to insulate, but obviously still needs testing (don't do this at home, yet).

The hot plate (obviously) means the bed is too tall for the finder levelling, and therefore the bed likely requires a reduction in the Z axis of the print volume.

Not sure how to deal with the levelling problem, unless I can drive that with G28 then G29 (Marlinesque), but a print should be doable, using CURA?   

Is it as straight forward as dropping the "Gantry Height" in the "Machine Settings" with the "FlashforgeFinderIntegration", or dropping both "Gantry Height" and "Z (Height)"?  Or is it going to need a hack of the plugin code please?  

I assume, on a normal finder, the "Heate bed" and "Heated build volume" checkboxes are unused in any event?  There seems no means to set the "Heated build volume" for example.

Cheers,
A

**Comment by ronoaldo** [comment]
Hi @Bazmundi - sorry for the late reply. I did a small answer about the topic but somehow I forgot to click comment button!

I believe that you can indeed add the heated bed, but I am not sure if the Finder firmware will be able to control it. As for the increased volume, yes, raising a bit the 140mm limit on the Pinter Profile should give you some stretch for like a cm or so, and the generated g-Code should allow you to print, but again, the commands to control the bed temperature must be supported by the firmware which I'm not sure Finder will do.

**Reply by Bazmundi** [reply]
@ronoaldo I do understand that the Finder firmware will not control the heated bed.  So, the problem is really just dealing with the fact my bed is about 1cm (ish) higher, due to the means I am using to attach it to the Finder.  I have a separate controller for the heatbed, so it would be controlled by the firmware.

**Comment by ronoaldo** [comment]
I'll move this to a discussion as of now, if we find something that can be changed in the plugin to support the heated bed then we add that back as a feature request.
